tests/test_notifier.py

```diff
@@ -335,3 +337,16 @@ def test_error_notifications_disabled():
 
     notice = future.result()
     assert notice["error"] == "error notifications are disabled"
+
+
+def test_notifier_with_backlog():
+    notifier = Notifier(test_backlog_enabled=True)
+
+    err = get_nested_exception()
+    notice = notifier.build_notice(err)
+
+    data = jsonify_notice(notice)
+
+    notifier._backlog.append_stats(data)
+
+    assert len(notifier._backlog._backlog) == 0
```

Comment:
Can you please explain what this assertion does?

Reply:
As I add data at [L350](https://github.com/airbrake/pybrake/blob/testcase_backlog/tests/test_notifier.py#L350), a backlog thread will run, pop up the data, and clear the backlog queue. After popping the data, it will break the loop and shut down the thread because it is enabled for test cases.

Reply:
Is there a way to assert that the data was, indeed, added? If `append_stats` doesn't append, then `notifier._backlog._backlog` is still 0.